app/src/main/res/drawable/custom_button_disabled.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
     android:shape="rectangle">
-    <solid android:color="@android:color/darker_gray"/>
+    <solid android:color="#443A64"/>
```

Comment:
Can this be consted to some sort of a resource file for colors?

Reply:
done